src/node_contextify.h

```diff
@@ -9,6 +9,13 @@
 namespace node {
 namespace contextify {
 
+struct ContextOptions {
+  v8::Local<v8::String> name;
+  v8::Local<v8::String> origin;
+  bool allowCodeGenStrings;
+  v8::Local<v8::Boolean> allowCodeGenWasm;
```

Comment:
snake_case

This mixing of `bool` and `v8::Boolean` is kind of weird although I understand why it is that way.